src/components/home/questionnaire-card.tsx

```diff
@@ -18,7 +19,7 @@ const QuestionnaireCard = () => {
       return (
         <CardContent>
           <Typography variant="h6" fontWeight={"bold"} style={{ marginTop: 6, marginBottom: 3 }}>
-            {strings.questionnaireCard.questionnairesBuilder}
+            {strings.questionnaireCard?.questionnairesBuilder}
```

Comment:
This is still using optional syntax- the questionnaireCard?.questionnairesBuilder- the ?.

Is there a reason for this, otherwise it should be removed.